### PR TITLE
Dompurify version update to 2.4.4

### PR DESCRIPTION
**Pull request by Ricokola**
Update Dompurify to version 2.4.4 to be up to date with GCWeb.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "bootstrap-sass": "3.4.1",
     "code-prettify": "^0.1.0",
     "datatables.net": "1.11.4",
-    "dompurify": "^2.4.0",
+    "dompurify": "^2.4.3",
```
Comment (duboisp) [inline package.json:45]: This version don't match what we do have in the package-lock.json.

**Review by duboisp — CHANGES_REQUESTED** [review]

**Comment by duboisp** [comment]
Pre-approved when the requested change are completed

**Comment by Ricokola** [comment]
@duboisp The change has been made, can you please validate and approve.

**Review by duboisp — APPROVED** [review]
Tested locally and work as expected.

**Pull request merged**